### DISCUSSION TITLE
build.sh language server

### DISCUSSION THREAD

**Discussion by Freed-Wu** [termux/termux-packages#17052] [Ideas]
I create [a language server](https://github.com/Freed-Wu/termux-language-server) for termux's `build.sh`. Can it be added to Wiki?

**Comment by agnostic-apollo** [comment]
Someone else would have to verify it's working. Probably can be added at https://wiki.termux.com/wiki/Building_packages

**Reply by Freed-Wu** [reply]
> Someone else would have to verify it's working

Welcome to verify. Some document to configure it is [here](https://termux-language-server.readthedocs.io/en/latest/resources/configure.html)
